spec/dom-elements-pool-spec.coffee

```diff
@@ -0,0 +1,41 @@
+DomElementsPool = require '../src/dom-elements-pool'
+
+describe "DomElementsPool", ->
```

Comment:
@lee-dohm: sounds good! Thank you for the suggestions, will add more specs shortly.